bettercli/cl/date.py

```
@@ -0,0 +1,316 @@
+import keyboard
+from logging import getLogger
+from .ansi import Cursor, ANSIColors
+
+logger = getLogger("bettercli.cl")
+
+T = t.TypeVar("T")
+LEN = t.TypeVar("LEN", bound=int, default=int)
+L = t.TypeVar("L", bound=list)
+
+class LenList(t.Generic[L, LEN], list):
+    def __init__(self, length:'LEN', list:'L'=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)